vnext/Shared/WinRTWebSocketResource.cpp

```diff
@@ -128,12 +131,16 @@ IAsyncAction WinRTWebSocketResource::PerformConnect() noexcept {
     } else {
       if (self->m_errorHandler) {
         auto error = winrt::hresult_error(result, winrt::hresult_error::from_abi);
-        self->m_errorHandler({winrt::to_string(error.message()), ErrorType::Connection});
+        std::stringstream hexCode;
```

Comment:
I think it would be nice to have this as a small helper functino, i.e.: `handleError(self, error, ErrorType::XYZ)`